components/org.wso2.carbon.security.sts.common/src/main/java/org/wso2/carbon/identity/sts/common/util/ServicePasswordCallbackHandler.java

```diff
@@ -78,6 +84,13 @@ public ServicePasswordCallbackHandler(SecurityConfigParams configParams, String
         this.serviceGroupId = serviceGroupId;
         this.realm = realm;
         this.configParams = configParams;
+        IdentityUtil.populateProperties();
+        // If the property is not available, default value is true.
+        this.includeUserStoreDomainInUsername = IdentityUtil.getProperty(INCLUDE_USER_STORE_DOMAIN_IN_USERNAME) == null
```

Comment:
Regarding `includeUserStoreDomainInUsername`, the null check has been used to make the default value `true` if the config did not exist in the identity.xml. This is because in current behavior we always include the user store domain in the local subject identifier. 

Regarding `includeTenantInUsername`, the current behavior does not include the tenant domain in the local subject identifier. Therefore the default value should be `false` and the null check is redundant. It is resolved with f05f605a90aa3f48ad59fb18cb22c7694e5c6011.